app/(api)/_datalib/media/update.ts

```diff
@@ -1,26 +1,23 @@
 import { getClient } from '@utils/mongodb/mongoClient.mjs';
 import { NotFoundError } from '@utils/response/Errors';
 
-export async function findCollectionItems(collection: never, query = null) {
+export async function updateMediaItem(id: string, inputData: string) {
+  // TODO: ? type of id
   try {
     const client = await getClient();
     const db = client.db;
-    const reqCollection = await db.getCollection(collection);
-    const reqDocument = await reqCollection.find(query).toArray();
+    const reqCollection = await db.getCollection('media');
+    const reqDocument = await reqCollection.updateOne(id, inputData);
     if (!reqDocument || reqDocument.length === 0) {
-      return {
-        ok: false,
-        body: null,
-        error: { code: 404, message: 'No items found in collection' },
-      };
+      throw new NotFoundError(`No Item Found.`);
     }
     return { ok: true, body: reqDocument, error: null };
   } catch (error) {
     if (error instanceof NotFoundError) {
```

Comment:
You won't need if statements in the catch block if you do the catch all system I described before